### DISCUSSION TITLE
Option to have dependency in multiple groups

### DISCUSSION THREAD

**Discussion by finswimmer** [renovatebot/renovate#32120] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab
Renovate: 38.128.6

### Please tell us more about your question or problem

Hey,

it looks like it is not possible that a dependency appears in multiple groups. Instead it seems, if group definition have an overlap, the later one wins.

What I would like to achieve is, that renovate creates one PR with all updates available (that's possible by using the preset `group:all`) but also creates one PR for each configured group. Doing this I can decide whether I want to update all deps at once or only some of the groups.

Is this somehow possible?

fin swimmer

**Comment by rarkins** [comment, marked as answer]
It's not possible

**Reply by finswimmer** [reply]
Thanks for your quick answer @rarkins.

Any chance that this would be accepted as a feature request?

**Reply by rarkins** [reply]
Yes, however it might be quite a complicated change so remain indefinitely open. Please start with a "Suggest an idea" Discussion describing how you think this could work (keeping in mind that most won't want this and we will keep default behavior)
